Sources_v3/StreamChatUI/ChatChannel/ChatMessageBubbleView.swift

```diff
@@ -41,6 +41,8 @@ open class ChatMessageBubbleView<ExtraData: UIExtraDataTypes>: View, UIConfigPro
     
     public private(set) lazy var borderLayer = CAShapeLayer()
 
+    private var layoutConstraints: [LayoutOptions: [NSLayoutConstraint]] = [:]
+
```

Comment:
I'm confused. I thought we'd use `ContainerStackView` for things like this. Why did you decide to go with constraints directly?

Reply:
The main reason is that with stack-view based cells the message list behaves much more glitchy while scrolling. 

Another reason is related to cell size calculations: when the photo gallery is shown, the cell should be of it's max possible size. If the gallery is hidden - the size should depend on other content. I was expecting it working out-of-the-box just manipulating `gallery.isHidden` but it occurs that hidden views constraints affect stack-view size. Changing `constraint.isActive` didn't help and the workarounds were ugly.